keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -32,8 +32,8 @@ class DeliveryConfigController(
     consumes = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE],
     produces = [APPLICATION_JSON_VALUE, APPLICATION_YAML_VALUE]
   )
```

Comment:
I'm confused how this one api change led to so many test changes?

Reply:
Oh you just updated the function signature for the test helpers, right?

Reply:
I just realized when fixing this that, for tests, it doesn't matter where we get the service account from when we're mocking the calls to allow/deny authorization, so I just removed the parameter from the test helper. Sorry if it was confusing, but the tests are a bit simpler for it.